tests/system/python/e2e/test_e2e_coap_pi_web_api.py

```diff
@@ -53,7 +53,7 @@ def _verify_egress(read_data_from_pi, pi_host, pi_admin, pi_passwd, pi_db, wait_
     data_from_pi = None
 
     # See C/plugins/common/omf.cpp
-    af_hierarchy_level = "foglamp_data_piwebapi"
+    af_hierarchy_level = "data_piwebapi"
```

Comment:
`foglamp/data_piwebapi` and treat the same way as for tests/system/python/packages/test_pi_webapi.py